app/containers/Achievements/LevelAchievement.jsx

```diff
@@ -2,17 +2,27 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
-import achievementBronze from 'images/achievement_bronze.svg?inline';
-import achievementSilver from 'images/achievement_silver.svg?inline';
-import achievementGold from 'images/achievement_gold.svg?inline';
-import achievementNotReached from 'images/achievement_not_reached.svg?inline';
+import achievementNotReached from 'images/achievement_not_reached.svg?external';
+import achievementReached from 'images/achievement_reached.svg?external';
 
 import Span from 'components/Span';
+import Icon from 'components/Icon';
 
-const Img = styled('img')`
+const BageIcon = styled(Icon)`
   margin-right: 15px;
 `;
 
+const LevelIcon = styled(BageIcon)`
+  .crown {
+    fill: ${props => {
+      if (props.level === 'bronze') return '#8b4513';
```

Comment:
Strings 'bronze', 'silver' and 'gold' can be moved to constants.

Reply:
I don't think, that is good idea, because these constants a defined as expressions, that depends on level numbers, that I get from component props.